src/test/java/ladder/domain/LadderTest.java

```diff
@@ -1,99 +1,98 @@
+            .as("값이 같고, 인스턴스도 같은 Line 이 존재할때 True 를 반환한다")
+            .isTrue(),
+        () -> assertThat(ladder.existLine(newLine11))
+            .as("값은 같지만, 인스턴스가 다른 Line 이 존재할때 True 를 반환한다")
+            .isTrue(),
+        () -> assertThat(ladder.existLine(newLine22))
+            .as("값은 같지만, 인스턴스가 다른 Line 이 존재할때 True 를 반환한다")
+            .isTrue(),
+        () -> assertThat(ladder.existLine(line33))
+            .as("값이 다른 인스턴스는 False 를 반환한다")
+            .isFalse()
+    );
+  }
 
-    @DisplayName("사다리 생성가능 여부를 판단하는 메서드를 검증한다")
-    @Test
-    public void canMakeLine() {
-        //given
-        Line lineA = new Line(11, 11);
-        Line lineB = new Line(6, 6);
-        Ladder ladder = Ladder.of(Set.of(lineA, lineB));
+  @DisplayName("사다리 생성가능 여부를 판단하는 메서드를 검증한다")
+  @Test
+  public void canMakeLine() {
+    //given
+    Line lineA = new Line(11, 11);
+    Line lineB = new Line(6, 6);
+    Ladder ladder = Ladder.of(Set.of(lineA, lineB), 12, 12);
 
-        Line notNearA = new Line(3, 3);
-        Line nearA = new Line(10, 11);
-        Line nearB1 = new Line(7, 6);
-        Line nearB2 = new Line(5, 6);
-        Line notNearB1 = new Line(6, 7);
-        Line notNearB2 = new Line(6, 5);
-        //when
-        //then
-        assertAll("Lines 가 입력한 파라미터에 맞게 생성된다",//isExistSameColumnAndAdjacentRow
-                () -> assertThat(ladder.hasCrossIntersection(notNearA))
-                        .as("notNearA 은 SameColumn && AdjacentRow 에 위치하지 않는다")
-                        .isFalse(),
-                () -> assertThat(ladder.hasCrossIntersection(nearA))
-                        .as("nearA 은 SameColumn && AdjacentRow 에 위치한다")
-                        .isTrue(),
-                () -> assertThat(ladder.hasCrossIntersection(nearB1))
-                        .as("nearB1 은 SameColumn && AdjacentRow 에 위치한다")
-                        .isTrue(),
-                () -> assertThat(ladder.hasCrossIntersection(nearB2))
-                        .as("nearB2 은 SameColumn && AdjacentRow 에 위치한다")
-                        .isTrue(),
-                () -> assertThat(ladder.hasCrossIntersection(notNearB1))
-                        .as("line33 은 SameColumn && AdjacentRow 에 위치한다")
-                        .isFalse(),
-                () -> assertThat(ladder.hasCrossIntersection(notNearB2))
-                        .as("line33 은 SameColumn && AdjacentRow 에 위치한다")
-                        .isFalse()
-        );
-    }
-}
\ No newline at end of file
+    Line notNearA = new Line(3, 3);
+    Line nearA = new Line(10, 11);
+    Line nearB1 = new Line(7, 6);
+    Line nearB2 = new Line(5, 6);
+    Line notNearB1 = new Line(6, 7);
+    Line notNearB2 = new Line(6, 5);
+    //when
+    //then
+    assertAll("Lines 가 입력한 파라미터에 맞게 생성된다",
+        () -> assertThat(ladder.hasCrossIntersection(notNearA))
+            .as("notNearA 은 SameColumn && AdjacentRow 에 위치하지 않는다")
+            .isFalse(),
+        () -> assertThat(ladder.hasCrossIntersection(nearA))
```

Comment:
```suggestion
        () -> assertThat(ladder.hasCrossIntersection(new Line(10, 11)))
```
굳이 별도의 로컬 변수로 분리하지 말고 직접 생성해도 되지 않을까?

Reply:
넵 불필요한 로컬변수를 분리하지않고 바로 생성하는 방향으로 수정하겠습니다!